ArduCopter/GCS_MAVLink_Copter.cpp

```diff
@@ -1099,6 +1100,10 @@ void GCS_MAVLINK_Copter::handle_message_set_position_target_local_ned(const mavl
         // rotate to body-frame if necessary
         if (packet.coordinate_frame == MAV_FRAME_BODY_NED || packet.coordinate_frame == MAV_FRAME_BODY_OFFSET_NED) {
             vel_neu_ms.xy() = copter.ahrs.body_to_earth2D(vel_neu_ms.xy());
+        } else if (packet.coordinate_frame == MAV_FRAME_BODY_FRD) {
+            vel_neu_ms.z = -vel_neu_ms.z;
```

Comment:
redundant.

Reply:
hi @peterbarker thank you for reviewing. I’m not sure I fully understand what you mean.  vel_neu_ms.z is negated because it will be multiplied by the body-to-NED rotation matrix, where the body frame is expected to be in FRD.